modules/gdpr_dump/plugins/gdpr_sanitizer/GDPRSanitizerDate.class.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Base class for export UI.
+ */
+class GDPRSanitizerDate extends GDPRSanitizerDefault {
+
+  /**
+   * {@inheritdoc}
+   */
+  public $name = 'gdpr_sanitizer_date';
+
+  /**
+   * {@inheritdoc}
+   */
+  public $label = 'Date sanitizer';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function sanitize($input, $field = NULL) {
+    $date = new DateTime('1000-01-01');
```

Comment:
I think this should be configurable so you can preserve date ranges (e.g. year or quarter, month etc)?